### PR TITLE
Fix misspellings in many AGE files

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/age/actions/runs/5251076994#summary-14208871240

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/age/actions/runs/5251077113#summary-14208871486

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ We strongly recommend you to subscribe the mailing lists, join the Apache AGE Di
 
 ## Pull Requests
 
-Changes to AGE source code are proposed, reviewed, and committed via Github pull requests (described in Code Convention). Pull requests are a great way to get your ideas into this repository. Anyone can view and comment on active changes here. Reviewing others' changes are a good way to learn how the change process works and gain exposure to activity in various parts of the code. You can help by reviewing the changes, asking questions, or pointing out issues as simple as typos.
+Changes to AGE source code are proposed, reviewed, and committed via GitHub pull requests (described in Code Convention). Pull requests are a great way to get your ideas into this repository. Anyone can view and comment on active changes here. Reviewing others' changes are a good way to learn how the change process works and gain exposure to activity in various parts of the code. You can help by reviewing the changes, asking questions, or pointing out issues as simple as typos.
```
Comment (jsoref) [inline CONTRIBUTING.md:23]: brand

File: README.md [hunk 2]
```diff
@@ -128,7 +128,7 @@ sudo apt-get install build-essential libreadline-dev zlib1g-dev flex bison
 Apache AGE is intended to be simple to install and run. It can be installed with Docker and other traditional ways. 
 
 <h4><a><img width="20" src="/img/pg.svg"></a>
-&nbsp;Install PosgtreSQL
+&nbsp;Install PostgreSQL
```
Comment (jsoref) [inline README.md:131]: brand

File: drivers/golang/install.sh [hunk 3]
```diff
@@ -104,7 +104,7 @@ if ! command -v go >/dev/null 2>&1 || { [ $(go version | grep -o -E '[0-9]+\.[0-
 	 		sudo open -w golang.pkg
 		  
 	  elif [[ "$os" == "Linux" ]]; then
-		  mdkir -p ~/tmp/go1.20.2
+		  mkdir -p ~/tmp/go1.20.2
```
Comment (jsoref) [inline drivers/golang/install.sh:107]: build failure

File: src/backend/executor/cypher_create.c [hunk 4]
```diff
@@ -283,7 +283,7 @@ static void end_cypher_create(CustomScanState *node)
 static void rescan_cypher_create(CustomScanState *node)
 {
     ereport(ERROR, (errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
-                    errmsg("cypher create clause cannot be rescaned"),
+                    errmsg("cypher create clause cannot be rescanned"),
```
Comment (jsoref) [inline src/backend/executor/cypher_create.c:286]: api change?

File: regress/expected/expr.out [hunk 5]
```diff
@@ -1441,9 +1441,9 @@ RETURN ''::int
 $$) AS r(result agtype);
 ERROR:  invalid input syntax for integer: ""
 SELECT * FROM cypher('expr', $$
-RETURN 'falze'::int
+RETURN 'false_'::int
```
Comment (jsoref) [inline regress/expected/expr.out:1444]: I'm hoping this is an acceptable change. While `falze` != `false`, `false_` != `false` too, and it has the benefit of not being flagged by spell checking tools.

I'm happy to drop this change or any other change.

File: regress/expected/index.out [hunk 6]
```diff
@@ -254,7 +254,7 @@ SELECT * FROM cypher('cypher_index', $$
         (us)<-[:has_city]-(:City {city_id: 3, name:"Los Angeles", west_coast: true, country_code:"US"}),
         (us)<-[:has_city]-(:City {city_id: 4, name:"Seattle", west_coast: true, country_code:"US"}),
         (ca)<-[:has_city]-(:City {city_id: 5, name:"Vancouver", west_coast: true, country_code:"CA"}),
-        (ca)<-[:has_city]-(:City {city_id: 6, name:"Toroto", west_coast: false, country_code:"CA"}),
+        (ca)<-[:has_city]-(:City {city_id: 6, name:"Toronto", west_coast: false, country_code:"CA"}),
```
Comment (jsoref) [inline regress/expected/index.out:257]: I live here.

File: src/backend/parser/cypher_expr.c [hunk 7]
```diff
@@ -153,15 +153,15 @@ static Node *transform_cypher_expr_recurse(cypher_parsestate *cpstate,
     case T_NullTest:
     {
         NullTest *n = (NullTest *)expr;
-        NullTest *tranformed_expr = makeNode(NullTest);
+        NullTest *transformed_expr = makeNode(NullTest);
 
-        tranformed_expr->arg = (Expr *)transform_cypher_expr_recurse(cpstate,
+        transformed_expr->arg = (Expr *)transform_cypher_expr_recurse(cpstate,
                                                          (Node *)n->arg);
```
Comment (jsoref) [inline src/backend/parser/cypher_expr.c:159]: I'n not sure about the whitespace here, but it doesn't seem to be particularly aligned, so I'm not touching it.

File: src/backend/utils/adt/age_global_graph.c [hunk 8]
```diff
@@ -41,7 +41,7 @@
 
 /* internal data structures implementation */
 
-/* vertex entry for the vertex_hastable */
+/* vertex entry for the vertex_hashtable */
```
Comment (jsoref) [inline src/backend/utils/adt/age_global_graph.c:44]: ?

**Review by jsoref — COMMENTED** [review]
Most corrections automatically suggested by Google Sheets. All fault mine

File: src/backend/utils/adt/age_global_graph.c (hunk 8 above)
Comment (muhammadshoaib) [reply src/backend/utils/adt/age_global_graph.c:44]: This one need to be checked in source code as well.

**Comment by dehowef** [comment]
The changes that affect spellings in actual code and not just comments might have had an issue, but all regression tests passed, and I reviewed all of the changes manually myself. It looks good, but can you squash all the commits down into one commit?

**Comment by jsoref** [comment]
@dehowef squashed

**Comment by muhammadshoaib** [comment]
Looks good to me.

**Comment by jrgemignani** [comment]
@jsoref Please squash your comments to something more descriptive without listing all of the words fixed. 

For example, for the commit description, something like "Fix misspellings in many AGE files (#985)"

For example, for the more detailed descriptions, mention what was done (a general description) and why. But, you don't need to list all of the individual changes. Just a good summary.

**Comment by jrgemignani** [comment]
@jsoref @dehowef @muhammadshoaib There are code reviews that need to be resolved.

**Comment by jrgemignani** [comment]
@jsoref  -
> For example, for the more detailed descriptions, mention what was done (a general description) and why. But, you don't need to list all of the individual changes. Just a good summary.

image

**Review by dehowef — COMMENTED** [review]
Changes look good, but please change the description of the commit to a brief summary

**Comment by jrgemignani** [comment]
@jsoref Please see above comments that are still waiting for you.

**Comment by jsoref** [comment]
@jrgemignani is this good enough?

**Pull request merged**